core/src/transaction.rs

```diff
@@ -240,6 +240,7 @@ pub struct LocalizedTransaction {
 impl LocalizedTransaction {
     /// Returns transaction signer.
     /// Panics if `LocalizedTransaction` is constructed using invalid `UnverifiedTransaction`.
+
```

Comment:
Could you remove this change?

This is the only change in the file but unnecessary.